source/Octopus.Tentacle/Kubernetes/KubernetesScriptPodCreator.cs

```diff
@@ -419,6 +415,25 @@ V1Affinity ParseScriptPodAffinity(InMemoryTentacleScriptLog tentacleScriptLog)
                 KubernetesConfig.PodAnnotationsJsonVariableName,
                 "pod annotations");
 
+        Dictionary<string, string>? GetScriptPodLabels(InMemoryTentacleScriptLog tentacleScriptLog, StartKubernetesScriptCommandV1 command)
+        {
+            var labels = new Dictionary<string, string>
+            {
+                ["octopus.com/serverTaskId"] = command.TaskId,
+                ["octopus.com/scriptTicketId"] = command.ScriptTicket.TaskId
+            };
+            var extraLabels = ParseScriptPodJson<Dictionary<string, string>>(
+                tentacleScriptLog,
+                KubernetesConfig.PodLabelsJson,
+                KubernetesConfig.PodLabelsJsonVariableName,
+                "pod labels");
+            labels.AddRange(extraLabels);
```

Comment:
I'm assuming that an empty env var results in an empty list that all just get's ignored?

Reply:
Yep - that's what happens in the annotations code at least 😅

Reply:
I thought that `ParseScriptPodJson` returned an empty, but instantiated dict. That should end up with just the source, right?

Reply:
Never mind, it does return null, but we're safe because our addRange allows for it. For now, I've added a gate for null just in case we ever change that implementation.